docs/sources/panels-visualizations/configure-thresholds/index.md

```diff
@@ -20,66 +20,104 @@ weight: 100
 - **Percentage** thresholds are defined relative to minimum or maximum. For example, 80 percent.
 
 ### Show threshold
 
-TBA
+{{< admonition type="note" >}}
+This option is supported for the bar chart, candlestick, time series, and trend visualizations.
+{{< /admonition>}}
 
-## Add a threshold
+Set if and how thresholds are shown with the following options.
 
-You can add as many thresholds to a panel as you want. Grafana automatically sorts thresholds values from highest to lowest.
+| Option                               | Example                                                                                                                                                                                          |
+| ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Off                                  |                                                                                                                                                                                                  |
+| As lines                             | {{< figure max-width="500px" class="w-100p" link-class="w-fit mx-auto d-flex flex-direction-column" src="screenshot-thresholds-lines-v10.4.png" alt="Visualization with threshold as a line" >}} |
+| As lines (dashed)                    | {{< figure max-width="500px" src="screenshot-thresholds-dashed-lines-v10.4.png" alt="Visualization with threshold as a dashed line" >}}                                                          |
+| As filled regions                    | {{< figure max-width="500px" src="screenshot-thresholds-regions-v10.4.png" alt="Visualization with threshold as a region" >}}                                                                    |
+| As filled regions and lines          | {{< figure max-width="500px" src="screenshot-thresholds-lines-regions-v10.4.png" alt="Visualization with threshold as a region and line" >}}                                                     |
+| As filled regions and lines (dashed) | {{< figure max-width="500px" src="screenshot-thresholds-dashed-lines-regions-v10.4.png" alt="Visualization with threshold as a region and dashed line" >}}                                       |
 
-To add a threshold:
+## Add a threshold
 
-1. Edit the panel to which you want to add a threshold.
+You can add as many thresholds to a visualization as you want. Grafana automatically sorts thresholds values from highest to lowest.
 
-1. In the options side pane, locate the **Thresholds** section and click **+ Add threshold**.
+1. Navigate to the panel you want to update.
+1. Hover over any part of the panel you want to work on to display the menu on the top right corner.
+1. Click the menu and select **Edit**.
+1. Scroll to the **Thresholds** section of the panel edit pane.
```

Comment:
as this section is at the end of the panel options it might be good to also suggest alternative of searching panel options for "thresholds"

<img width="582" alt="Screenshot 2024-01-31 at 2 02 18 PM" src="https://github.com/grafana/grafana/assets/22381771/27ccef0c-1083-4591-aa07-820492b88d6e">

Reply:
Good call, will do.